src/java.base/share/classes/jdk/internal/foreign/abi/ppc64/sysv/SysVPPC64leVaList.java

```diff
@@ -43,6 +43,7 @@
 import jdk.internal.foreign.abi.SharedUtils;
 import jdk.internal.foreign.abi.SharedUtils.SimpleVaArg;
 import jdk.internal.foreign.MemorySessionImpl;
+import static jdk.internal.foreign.abi.SharedUtils.THROWING_ALLOCATOR;
```

Comment:
Please remove this line so the change on line 107 is not needed.

Reply:
Removed as suggested above.